FruitFruit/FruitFruit/Models/FruitStatus.swift

```diff
@@ -14,7 +14,7 @@ enum FruitStatus: String {
     case Arriving = "Arriving"
     case Arrived = "Arrived"
     
-    func homeTitleText(fruit: String, time: Int, place: String) -> NSAttributedString  {
+    func homeTitleText(fruit: String, time: Int, place: String) -> NSMutableAttributedString  {
```

Comment:
```suggestion
    func makeHomeTitleText(fruit: String, time: Int, place: String) -> NSMutableAttributedString  {
```

method 이름은 동사로 시작되어야해요